### PR TITLE
feat: extend initPolkadotSnap params

### LINKED ISSUE #204: Call initiatePolkadotSnap, not enablePolkadotSnap, key is `Snap`, not `metamask-polkadot-snap`.
The README -> Integration steps are stating to call enablePolkadotSnap, but to get it into `injectedWeb3`, initiatePolkadotSnap needs to be called instead - and there is no arg option to pass a config to it.

I have also noticed the key once injected is simply `Snap`, not `metamask-polkadot-snap` (see [this](https://github.com/ChainSafe/metamask-snap-polkadot/blob/e0f3d4fc0be7366c62211e29d3a276e4fab5669e/packages/adapter/src/extension/index.ts#L25))

### PR DESCRIPTION
**Pull request by irubido**
option to change params when using initPolkadotSnap for substrate dapps

Closes #204

### CODE REVIEW

File: packages/adapter/README.md [hunk 1]
```diff
@@ -4,14 +4,12 @@
 ![](https://img.shields.io/badge/yarn-%3E%3D1.17.0-orange.svg?style=flat-square)
 ![Discord](https://img.shields.io/discord/608204864593461248?color=blue&label=Discord&logo=discord)
 
-Metamask <> Polkadot snap adapter is used to inject [polkadot snap](https://github.com/chainsafe/metamask-snap-polkadot) as web3 provider. It lists snap inside `window.injectedWeb3["metamask-polkadot-snap"]` so it can be enabled using `@polkadot/extension-dapp` package.  
+Metamask <> Polkadot snap adapter is used to inject [polkadot snap](https://github.com/chainsafe/metamask-snap-polkadot) as web3 provider. It lists snap inside `window.injectedWeb3["Snap"]` so it can be enabled using `@polkadot/extension-dapp` package.
```
Comment (mpetrunic) [inline packages/adapter/README.md:7]: maybe we should allow users to configure injected key? Not sure Snap is most sane default, we could cut breaking change to adapter to set more sane default like metamask-polkadot-snap ?

Comment (irubido) [reply packages/adapter/README.md:7]: Sounds good

**Review by Lykhoyda — APPROVED** [review]

**Comment by ekenigs** [comment]
@Lykhoyda @irubido when do you think you can release this?

**Comment by BeroBurny** [comment]
> @Lykhoyda @irubido when do you think you can release this?

Will need to do a few things before being able to release, but working on it to make it shortly possible

**Pull request merged**